File: java/socketd/src/main/java/org/noear/socketd/transport/core/Reply.java
package org.noear.socketd.transport.core;

/**
 * 答复实体
 *
 * @author noear
 * @since 2.1
 */
public interface Reply extends Entity {
    /**
     * 流Id
     */
    String sid();

    /**
     * 是否答复结束
     */
    boolean isEnd();
}

File: swift/socketd/socketd/src/socketd/transport/core/Reply.swift
//
//  Reply.swift
//  socketd
//
//  Created by noear on 2023/12/27.
//

import Foundation

/**
 * 答复实体
 *
 * @author noear
 * @since 2.1
 */
protocol Reply : Entity{
    /**
     * 流Id
     */
    func sid() -> String;
    
    /**
     * 是否答复结束
     */
    func isEnd() -> Bool;
}
